dotnet/frontend/Startup.cs

```diff
@@ -46,7 +45,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
             app.UseRouting();
 
-            app.UseEndpoints(endpoints => { endpoints.MapControllers(); });

```

Comment:
Editorconfig?

Reply:
This one isn't editorconfig-able.

The old code was creating a new statement scope (called block too) an running a statement within. It's subtly different semantics but produces the same output. And critically, not a style thing.

You can verify this with the following links:

[Sharplab - scope added](https://sharplab.io/#v2:D4AQTAjAsAUCDMACciDCiDetE+UkALIgLIA8AKgJIB2AfABQgRgU22ICGAxgC4CWAe2oBKTAF9suSTgTIiAEQECe9YdMzrcJUn2o8GAD0QBedhmQQAnPQPCA3IjGJ76iTDFA)
[Sharplab - no scope](https://sharplab.io/#v2:D4AQTAjAsAUCDMACciDCiDetE+UkALIgLIA8AKgJIB2AfABQgRgU22ICGAxgC4CWAe2oBKTAF9suSTgTIiAEQECe9YdMzrcJUn2o8GAD0QBedkwCc9A8OEBudRJhigA=)